packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -1,6 +1,51 @@
 {
     "version": 1,
-    "timestamp": "2021-05-01T00:00:00+00:00",
-    "sequence": 1,
-    "actions": []
+    "timestamp": "2021-06-09T12:00:00+00:00",
+    "sequence": 2,
+    "actions": [
+        {
+            "conditions": [
+                {
+                    "os": {
+                        "macos": "*",
+                        "linux": "!",
+                        "windows": "!",
+                        "android": "!",
+                        "ios": "!"
+                    },
+                    "environment": {
+                        "desktop": "!",
+                        "mobile": "!",
+                        "web": "*"
+                    },
+                    "browser": {
+                        "firefox": "!",
+                        "chrome": "91",
+                        "chromium": "!"
+                    },
+                    "transport": {
+                        "bridge": "!",
+                        "webusbplugin": "*"
+                    }
+                }
+            ],
+            "message": {
+                "id": "0f3ec64d-c3e4-4787-8106-162f3ac14c34",
+                "priority": 91,
+                "dismissible": true,
+                "variant": "critical",
+                "category": "banner",
+                "content": {
+                    "en-GB": "To be able to use the application in the current version of your browser, please install the Trezor Bridge."
```

Comment:
It might be a good idea to get eyes of some native speaker on this before we'll push it to all the chrome users. Like Anthony, for example.

Reply:
checked ✅